### DISCUSSION TITLE
please help

### DISCUSSION THREAD

**Discussion by inthepresent27** [ramensoftware/windhawk-mods#1256] [Q&A]
I have tried everything and watched over 10 videos on YouTube trying to fix this issue, I can not install it, it keeps stopping incomplete with the 1053 error![Screenshot 2024-11-17 051131](https://github.com/user-attachments/assets/91f82fa7-0d5e-4484-9855-342bb730cc0c)

**Comment by m417z** [comment]
Most likely you're using Windows 11 on ARM which is not supported, see https://github.com/ramensoftware/windhawk/issues/11.

**Reply by inthepresent27** [reply]
Device name	JoshuaTree
Processor	Snapdragon(R) X 10-core X1P64100 @ 3.40 GHz   3.42 GHz
Installed RAM	16.0 GB (15.6 GB usable)
System type	64-bit operating system, ARM-based processor
Pen and touch	Pen and touch support with 10 touch points

thank you, I was going crazy, I love the work, I have it  on an AMD RYZEN, and it looks beautiful
